product_docs/docs/biganimal/release/free_trial/detail/create_an_account/index.mdx

```diff
@@ -4,17 +4,18 @@ description: "Creating an EDB account and signing into the BigAnimal free trial
 navTitle: "Create an account" 
 ---
 
-Registering an account on [enterprisedb.com](https://www.enterprisedb.com/) gets you access to the BigAnimal trial, as well as EDB's training and software downloads. 
+You will need to register with EDB in order to access BigAnimal free trial, as well as EDB's training and software downloads.
 Accounts are free, and registration takes less than a minute.
 
 1. Navigate to <https://biganimal.com>.
-2. Select the **Try it for free** button.
-3. Fill in the form, and agree to the terms and conditions.
+2. Select the **Test drive for free** button.
+3. Fill in the form, and agree to the **Terms and Conditions** and the **Privacy Policy**.
```

Comment:
Select **Test drive for free**.
Fill in the form, and agree to the terms and conditions and privacy policy.